assets/js/components/adminbar/AdminBarSessions.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Admin Bar Sessions component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AdminBarPreview from './AdminBarPreview';
+import Data from 'googlesitekit-data';
+import DataBlock from '../data-block';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../modules/analytics/datastore/constants';
+import { changeToPercent, readableLargeNumber } from '../../util';
+const { useSelect } = Data;
+
+const AdminBarSessions = ( { classNames } ) => {
+	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+	const reportArgs = {
+		...dateRangeDates,
+		dimensions: 'ga:date',
+		limit: 10,
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Users',
+			},
+			{
+				expression: 'ga:sessions',
+				alias: 'Sessions',
+			},
+			{
+				expression: 'ga:bounceRate',
+				alias: 'Bounce Rate',
+			},
+			{
+				expression: 'ga:avgSessionDuration',
+				alias: 'Average Session Duration',
```

Comment:
@felixarntz – is there any reason to pass these aliases in the request? I don't think we use them at all – certainly not for display since they're not translated, but just curious if there is any practical reason for this other than maybe inspectability in the response?

If only for us – that's a perfectly valid reason but it's slightly misleading in that it's formatted in a way you might think it's intended to be a user-facing string.